### PR TITLE
feat(web): Add script to update workspace  RS version after release

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
 		"prepublish": "yarn run build",
 		"test": "jest --watch",
 		"typescript": "tsc -p tsconfig.json",
-		"build-example": "scripts/build-example.sh 3.0.0-rc.0"
+		"build-example": "scripts/build-example.sh 3.0.0-rc.3",
```
Comment (bietkul) [inline packages/web/package.json:25]: I think there should be some way to pick the version from version field of `package.json`.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/web/package.json (hunk 1 above)
Comment (jyash97) [reply packages/web/package.json:25]: Yeah that will be great, any idea or lead how this can be done ? 
I searched but didnt got a good way to do this.

Comment (bietkul) [reply packages/web/package.json:25]: Check this https://github.com/kentcdodds/nps
You can define the scripts outside of the package.json
Ref. https://github.com/appbaseio/appbase-js

Comment (jyash97) [reply packages/web/package.json:25]: Fixed in #959

**Pull request closed without merging**